mirage/factories/institution.ts

```diff
@@ -5,8 +5,9 @@ import Institution from 'ember-osf-web/models/institution';
 import { placekitten, randomGravatar } from '../utils';
 
 export interface InstitutionTraits {
-    withInstitutionalUsers: Trait;
-    withStatSummary: Trait;
+    withInstitutionDepartments: Trait;
+    withInstitutionUsers: Trait;
+    withSummaryMetrics: Trait;
```

Comment:
Make a single trait to handle all of the metrics stuff and have the summary metrics be accurate